src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/HandlerProvider.cs

```diff
@@ -25,15 +23,15 @@ public HandlerProvider(ILspServices lspServices)
     }
 
     /// <summary>
-    /// Get the MethodHandler for a particular request.
+    /// Get the <see cref="IMethodHandler"/>s for a particular request.
     /// </summary>
     /// <param name="method">The method name being made.</param>
     /// <param name="requestType">The requestType for this method.</param>
     /// <param name="responseType">The responseType for this method.</param>
     /// <returns>The handler for this request.</returns>
-    public IMethodHandler GetMethodHandler(string method, Type? requestType, Type? responseType)
+    public IMethodHandler GetMethodHandler(string method, Type? requestType, Type? responseType, string? language = null)
```

Comment:
def worried about the callers forgetting to pass this along.  can we make it required?